src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/TraceLogging/TraceLoggingEventSource.cs

```diff
@@ -38,7 +38,7 @@ public partial class EventSource
 #endif
 
 #if FEATURE_PERFTRACING
-        private readonly TraceLoggingEventHandleTable m_eventHandleTable = new TraceLoggingEventHandleTable();
+        private readonly TraceLoggingEventHandleTable m_eventHandleTable = null!;
```

Comment:
I don't understand this change.  Where is it initialized now?

Reply:
Nevermind, found it; it was hidden in the diff initially.

Reply:
The issue here is that `TraceLoggingEventHandleTable` can't get trimmed if it is unconditionally initialized. I couldn't think of a better way to do this nicely. I wanted to initialize it in the "Initialize" method, but then it couldn't be `readonly` anymore.